src/svgp.jl

```diff
@@ -1,10 +1,13 @@
-struct SVGP end
+struct SVGP{Tfz<:FiniteGP,Tq<:AbstractMvNormal}
+    fz::Tfz
+    q::Tq
+end
 
 raw"""
-    approx_posterior(::SVGP, fz::FiniteGP, q::MvNormal)
+    approx_posterior(svgp::SVGP)
 
-Compute the approximate posterior [1] over the process `f = fz.f`, given inducing
-inputs `z = fz.x` and a variational distribution over inducing points `q(u)` where `u =
+Compute the approximate posterior [1] over the process `f = svgp.fz.f`, given inducing
+inputs `z = svgp.fz.x` and a variational distribution over inducing points `svgp.q(u)` where `u =
```

Comment:
I find it a bit confusing to mix code (`svgp.q`) and maths (`q(u)`) ... maybe
```
`svgp.q` which represents :math:`q(u)` where ...
```
?